source/quakedef.h

```diff
@@ -214,7 +216,13 @@ extern int psp_system_model;
 #include "psp/common.h"
 #include "psp/vid.h"
 #include "psp/sys.h"
-#endif // _3DS
+#elif __WII__
+#include <ogcsys.h>
```

Comment:
`ogcsys.h` should go at the top if its totally necessary to be in quakedef and not somewhere else. But I'd recommend keeping quakedef clean of this type of stuff